packages/lab/src/stepped-tracker/SteppedTracker.css

```diff
@@ -1,17 +1,41 @@
 .saltSteppedTracker {
-  margin: 0;
-  padding: 0;
-  text-indent: 0;
-  list-style-type: none;
-  display: flex;
+  grid-area: SteppedTracker;
   width: 100%;
-  position: relative;
+  height: 100%;
+
+  padding: var(--salt-focused-outlineWidth);
+  margin: calc(-1 * var(--salt-focused-outlineWidth));
+  list-style-type: none;
+  box-sizing: content-box;
+
+  transition-duration: var(--salt-duration-perceptible);
+  transition-timing-function: ease-in-out;
+  transition-property: opacity, visibility;
+}
+
+@media (prefers-reduced-motion) {
+  .saltSteppedTracker {
+    transition-duration: var(--salt-duration-instant);
+  }
 }
 
-.saltSteppedTracker.saltSteppedTracker-horizontal {
+.saltSteppedTracker-horizontal {
+  display: flex;
   flex-direction: row;
 }
 
-.saltSteppedTracker.saltSteppedTracker-vertical {
+.saltSteppedTracker-vertical {
+  display: flex;
   flex-direction: column;
+  overflow: hidden;
+}
+
+.saltSteppedTracker-vertical > .saltStep.saltStep-expanded > .saltSteppedTracker {
+  opacity: 1;
+  visibility: visible;
+}
+
+.saltSteppedTracker-vertical > .saltStep.saltStep-collapsed > .saltSteppedTracker {
+  opacity: 0;
+  visibility: hidden;
```

Comment:
Should `visibility` be used or HTML `hidden`? Compare to accordion. cc @jake-costa

Reply:
@origami-z visibility: hidden is fine but aria-hidden and HTML hidden should be used on the nested ol. But I believe they are already like accordion. 

@chrispcode correct me if I am wrong and let me know once stepped tracker is ready for the accessibility validation review.

Reply:
Yes both hidden properties exist on the nested stepper, based on if it expanded or not! And yes will give you another heads up when I fix the last issue

Reply:
Done!